tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
```

Comment:
I have the feeling that this is for TypeScript, we shouldn’t need that yet, right?